### ISSUE TITLE
[BUG] I found the timer maybe not accurate.

### ISSUE THREAD

**Issue by FrancisChuh** [Nriver/tomato-timer-widget#3]
The timer is not accurate.

**Comment by djubre7** [comment]
Hey, I know this is late, but inside the config note there is a setting that you can tweak and set it right, or however you want.

This is the part of code you want to modify:
_const WORK_TIME = 1500; //1500s = 25min
const BREAK_TIME = 300; // 300s = 5min_

Here is the complete note: 

```

// Please do not modify the name of this note.

// Change language name to anyone you like in the translations file. For example, 'cn' for Chinese, 'en' for english.
var lang = 'en';
// Set true to enable sound effects, false to disable it.
var enableSound = true;
// Start count down automatically when you open Trilium.
var autoStartCountDown = true;

const WORK_TIME = 1500; //1500s = 25min
const BREAK_TIME = 300; // 300s = 5min

// Don't forget to expose configs to the main js file
module.exports = {
    enableSound,
    autoStartCountDown,
    lang,
    WORK_TIME,
    BREAK_TIME
}
```

**Comment by FrancisChuh** [comment]
> Hey, I know this is late, but inside the config note there is a setting that you can tweak and set it right, or however you want.
> 
> This is the part of code you want to modify: _const WORK_TIME = 1500; //1500s = 25min const BREAK_TIME = 300; // 300s = 5min_
> 
> Here is the complete note:
> 
> ```
> 
> // Please do not modify the name of this note.
> 
> // Change language name to anyone you like in the translations file. For example, 'cn' for Chinese, 'en' for english.
> var lang = 'en';
> // Set true to enable sound effects, false to disable it.
> var enableSound = true;
> // Start count down automatically when you open Trilium.
> var autoStartCountDown = true;
> 
> const WORK_TIME = 1500; //1500s = 25min
> const BREAK_TIME = 300; // 300s = 5min
> 
> // Don't forget to expose configs to the main js file
> module.exports = {
>     enableSound,
>     autoStartCountDown,
>     lang,
>     WORK_TIME,
>     BREAK_TIME
> }
> ```

Yes, I known this setting..But I mean the timer might be not accurate.. Especially running it background, could you check again from your side? Thanks 🙏

**Comment by FrancisChuh** [comment]
Yes, I known this setting..
But I mean the timer might be not accurate.. Especially running it background, could you check again from your side? 
Thanks
